lib/jsanalyze.js

```diff
@@ -230,9 +230,7 @@ exports.analyzeJs = function analyzeJs(contents, opts) {
 		retainLines: true,
 		presets: [],
 		plugins: [
-			[ require.resolve('./babel-plugins/global-scope'), {
-				logger: opts.logger
-			}]
+			[ require.resolve('./babel-plugins/global-scope'), { logger: opts.logger } ]
 		]
```

Comment:
To avoid adding the `"use strict"`, we also need to define it on the `parserOpts` passed to the `babel.transform` function like below

```
parserOpts: {
	sourceType: 'unambiguous'
}
```

Side note, I was reading the babel doc and it looks like I missed the note about transforms sync behaviour being deprecated and replaced with transformSync, if you're editing the code, could you edit the call to be `results.contents = babel.transformSync(contents, options).code;`